pages/desktop/consumer_pages/base.py

```diff
@@ -67,13 +70,22 @@ def set_region(self, region):
 
         self.wait_notification_box_not_visible()
 
-    def login(self, user=None):
-        from pages.fxa import FirefoxAccounts
-        fxa = FirefoxAccounts(self.testsetup)
-        fxa.login_user(user)
+    def login(self, email, password):
+        from fxapom.pages.desktop.sign_in import SignIn
+        fxa_login = SignIn(self.testsetup)
+        fxa_login.sign_in(email, password)
+
         self.wait_notification_box_visible()
         self.wait_notification_box_not_visible()
 
+    def create_new_user(self):
+        acct = FxATestAccount(use_prod=False).create_account()
+        return MockUser(email=acct.email, password=acct.password, name=acct.email.split('@')[0])
+
+    def get_default_user(self, user='default'):
```

Comment:
This seems a bit odd that the method is called `get_default_user`, and yet it accepts an argument for `user` which defaults to `default`. That would mean I could call `get_default_user('bob')` and it would return the credentials for `bob`, which seems odd because the name of the method is `get_default_user`. Maybe it should be renamed to simply `get_user`.